### ISSUE TITLE
CI failing

### ISSUE THREAD

**Issue by jfirebaugh** [mapbox/mapbox-gl-js#1767]
[CI started failing without any code changes](https://circleci.com/gh/mapbox/mapbox-gl-js/1499):

```
      test exited without ending
      Error: test exited without ending
          at process.emit (events.js:117:20)
```

Most likely a package updated from underneath us and either broke semver, has a bug, or we're using it incorrectly. We can compare version numbers from the [last good run](https://circleci.com/gh/mapbox/mapbox-gl-js/1489) and see what changed.

@lucaswoj can you track this down. If it is indeed a package breakage, we should start using shrinkwrap. This is the second or third time this has happened.

**Comment by lucaswoj** [comment]
On it @jfirebaugh

**Comment by lucaswoj** [comment]
While random CI failures feel unacceptable...

> This is the second or third time this has happened.

...this one turned out to be meaningful. We were failing our own `mapbox-gl-style-spec` validation during unit tests and ignoring the error event.

I'm weakly opposed to using shrinkwrap because 

 - we would be on the hook for remembering to update module versions periodically
 - this project is effectively shinkwrapped when we create dist builds 
 - we can individually pin problematic module's versions in `package.json` anyway

@jfirebaugh was there any pattern to the other CI failures?

**Comment by lucaswoj** [comment]
The failures are fixed in #1769 but lets leave this ticket open until we make a decision about shrinkwrap.

**Comment by jfirebaugh** [comment]
We can continue discussion in #1607. But I'm ok with the status quo for now.
